docs/install/install-edex.md

```diff
@@ -275,3 +274,114 @@ This file specifies configuration and runtime parameters. If you are pulling in
 ```
 rpm -qa | grep awips2-edex
 ```
+
```

Comment:
Add a section break `---` before the beginning of this new section "Uninstalling EDEX"